backend/src/schema/resolvers/notifications.js

```diff
@@ -48,16 +48,18 @@ export default {
       const readTxResultPromise = session.readTransaction(async transaction => {
         const notificationsTransactionResponse = await transaction.run(
           ` 
-          MATCH (resource {deleted: false, disabled: false})-[notification:NOTIFIED]->(user:User {id:$id})
-          ${whereClause}
-          WITH user, notification, resource,
-          [(resource)<-[:WROTE]-(author:User) | author {.*}] as authors,
-          [(resource)-[:COMMENTS]->(post:Post)<-[:WROTE]-(author:User) | post{.*, author: properties(author)} ] as posts
-          WITH resource, user, notification, authors, posts,
-          resource {.*, __typename: labels(resource)[0], author: authors[0], post: posts[0]} as finalResource
-          RETURN notification {.*, from: finalResource, to: properties(user)}
-          ${orderByClause}
-          ${offset} ${limit}
+            MATCH (resource)-[notification:NOTIFIED]->(user:User {id:$id})
+            WHERE ((labels(resource)[0] in ["Post", "Comment"] AND NOT resource.deleted AND NOT resource.disabled) OR labels(resource)[0] in ["Report"])
```

Comment:
I had to re-read this many times to understand what's going on. Could you put that on two lines to make it more obvious?
```
WHERE
  ((labels(resource)[0] in ["Post", "Comment"] AND NOT resource.deleted AND NOT resource.disabled)
  OR labels(resource)[0] in ["Report"])
```

Reply:
Yes, for sure!
I understand you. @roschaefer